<p>Hardcoded secrets can be abused to gain unauthorized access and compromise the security perimiter.</p>
<h2>Noncompliant Code Example</h2>
<pre>
TO DO
</pre>
<h2>Compliant Solution</h2>
<pre>
TO DO
</pre>