<p>Private keys exposed in code may lead to impersonation, data falsification and service compromise.</p>
<h2>Noncompliant Code Example</h2>
<pre>
TO DO
</pre>
<h2>Compliant Solution</h2>
<pre>
TO DO
</pre>
