<p>Hardcoded AWS secrets can be abused to gain unauthorized access to the shared cluster and/or the VPC.</p>
<h2>Noncompliant Code Example</h2>
<pre>
TO DO
</pre>
<h2>Compliant Solution</h2>
<pre>
TO DO
</pre>
